_events/talks/event-030.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Research Squirrel Engineers - An independent squirrel network for RSEs in DH and archaeology"
+authors:
+    - &speaker
+      name: Sophie Schmidt
+      orcid: 0000-0003-4696-2101
+      is_speaker: true
+      affiliation: 2
+    - name: Florian Thiery
+      orcid: 0000-0002-3246-3531
+      is_speaker: true
+      affiliation: 4
+    - name: Timo Homburg
+      orcid: 0000-0002-9499-5840
+      affiliation: 3
+    - name: Martina Trognitz
+      orcid: 0000-0003-0485-6861
+      affiliation: 1
+affiliations:
+    - name: Austrian Centre for Digital Humanities and Cultural Heritage (ACDH-CH), Austria
+      index: 1
+    - name: Deutsches Archäologisches Institut, Germany
+      index: 2
+    - name: Mainz University Of Applied Sciences, Germany
+      index: 3
+    - name: Research Squirrel Engineers, Germany
+      index: 4
+author: *speaker
+category: talks
+language: english
+date: 2020-11-03
+---
+People who write code for research are few and far between in the archaeological sciences. Quite often they are regarded as technicians and their function in research projects reduced to those of "helpers", diminishing their contribution to the success of the undertaking. Nowadays some students of the humanities are trained in digital technologies to bridge the information gap between the different fields (Digital Humanities, DH). A similar situation can be found  in Digital and Computational Archaeology, though the two disciplines only marginally influenced each other (Hugget 10.12759/hsr.37.2012.3.86-105). Some centers of Digital Archaeology work closely with DH departments and some DH specialists moved from archaeology to the broader field of DH, which shows that the job market in Digital Archaeology does not offer enough possibilities. The temporary nature of research related projects is another reason for an unstable job market in the field. To mitigate the effects of this instability, increased workload, surveillance and underappreciation, Hugget calls for a resilient scholarship in a digital age (Hugget 10.5334/jcaa.25). One important aspect he names on the individual level is forming a community for networking and support. We will present the "Research Squirrel Engineers", an open, diverse and international network, which aims at RSEs connecting and developing their own ideas for side projects independent from institutions (and funding). Within the Research Squirrel community, currently two members have an engineering background (computer science, geoinformatics) whereas the other two studied a humanities subject (archaeology). By focusing on their own research interests they help RSEs to retain joy in their job, diversify and showcase their skill sets. So far Squirrels focus on (Linked) Open Data projects, but are happy to receive suggestions and new research squirrels joining at SORSE.
```

Comment:
we should convert the DOIs here to links. I'd do this manually as this will likely not happen always. Ok @eileen-kuehn? 

```suggestion
People who write code for research are few and far between in the archaeological sciences. Quite often they are regarded as technicians and their function in research projects reduced to those of "helpers", diminishing their contribution to the success of the undertaking. Nowadays some students of the humanities are trained in digital technologies to bridge the information gap between the different fields (Digital Humanities, DH). A similar situation can be found  in Digital and Computational Archaeology, though the two disciplines only marginally influenced each other (Hugget [10.12759/hsr.37.2012.3.86-105](https://doi.org/10.12759/hsr.37.2012.3.86-105)). Some centers of Digital Archaeology work closely with DH departments and some DH specialists moved from archaeology to the broader field of DH, which shows that the job market in Digital Archaeology does not offer enough possibilities. The temporary nature of research related projects is another reason for an unstable job market in the field. To mitigate the effects of this instability, increased workload, surveillance and underappreciation, Hugget calls for a resilient scholarship in a digital age (Hugget [10.5334/jcaa.25](https://doi.org/10.5334/jcaa.25)). One important aspect he names on the individual level is forming a community for networking and support. We will present the "Research Squirrel Engineers", an open, diverse and international network, which aims at RSEs connecting and developing their own ideas for side projects independent from institutions (and funding). Within the Research Squirrel community, currently two members have an engineering background (computer science, geoinformatics) whereas the other two studied a humanities subject (archaeology). By focusing on their own research interests they help RSEs to retain joy in their job, diversify and showcase their skill sets. So far Squirrels focus on (Linked) Open Data projects, but are happy to receive suggestions and new research squirrels joining at SORSE.
```